ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -1486,6 +1504,24 @@ setOptions({
         }
     }
 });
+addEvent(H.Chart, 'render', function (): void {
```

Comment:
It would be nice to comment why this event is added or at least add a comment with a related ticket number only. Might be useful in the future